.teamcity/_self/projects/WPComPlugins.kt

```diff
@@ -29,7 +29,7 @@ object WPComPlugins : Project({
 	buildType(EditingToolkit)
 	buildType(WpcomBlockEditor)
 	buildType(Notifications)
-	buildType(CalypsoStats)
+	buildType(OdysseyStats)
```

Comment:
Definitely a good time to change the build, since it'd be harder to do that at a later time